src/js/background.js

```diff
@@ -776,6 +781,29 @@ function startBackgroundListeners() {
     }
   }, {urls: ["http://*/*", "https://*/*"]}, []);
 
+  if(badger.isFirefoxMobile) {
+    // keeps track of popup id while one is open
+    var openPopupId;
+
+    // fake a popup
+    chrome.browserAction.onClicked.addListener(() => {
+      chrome.tabs.query({active: true, lastFocusedWindow: true}, (tabs) => {
+        var url = chrome.runtime.getManifest().browser_action.default_popup + "?tabId=" + tabs[0].id;
+        chrome.tabs.create({url, index: tabs[0].index + 1}, (tab) => {
+          openPopupId = tab.id;
+        });
+      });
+    });
+
+    // remove the 'popup' when another tab is activated
+    chrome.tabs.onActivated.addListener((activeInfo) => {
+      if(openPopupId && openPopupId != activeInfo.tabId) {
```

Comment:
I think this will fail if `openPopupId = 0`. We can instead check for a sentinel value like:
```javascript
var openPopup = false;
...
if ((openPopupId != false) && ...) {
  ...
  openPopupId = false;
}
```

Also, I think there might be an issue here if the user changes the url of the popup tab to something else, then visits another tab. Would it close the tab that was formerly a popup? I'll try this out when I'm testing today.